acvm-repo/acvm/src/pwg/mod.rs

```diff
@@ -244,6 +256,24 @@ impl<'a, F: AcirField, B: BlackBoxFunctionSolver<F>> ACVM<'a, F, B> {
         self.profiling_active = profiling_active;
     }
 
+    // Enable brillig fuzzing
+    pub fn with_brillig_fuzzing(
+        &mut self,
+        brillig_fuzzing_active: bool,
+        brillig_branch_to_feature_map: Option<&'a BranchToFeatureMap>,
+    ) {
```

Comment:
This could be:

```suggestion
    pub fn with_brillig_fuzzing(
        &mut self,
        brillig_branch_to_feature_map: &'a BranchToFeatureMap,
    ) {
```

That if, when this is called it's because we want fuzzing active, and the map must be a `Some` value.

Then, maybe `brillig_fuzzing_active` isn't needed anymore, we could know if it's active or not based on whether the map is None or Some.

Reply:
Done